edit/settings.js

```diff
@@ -29,12 +30,30 @@ function StyleSettings(editor) {
     if (!newStyle.id) return;
     if (reason === 'editSave' || reason === 'config') return;
     style = newStyle;
-    document.querySelector('.style-settings').disabled = false;
+    $('.style-settings').disabled = false;
     inputs.forEach(i => i.update());
   }
 
+  function createArea([parentSel, type]) {
```

Comment:
Maybe we should use a different name since this function would only work with inclusion/exclusion. How about `createExclusionArea`?

Reply:
We can rename it afterwards. I don't like overly-specific names unless necessary.